lib/manager/components/reporter/components/RouteLayout.js

```diff
@@ -98,6 +98,20 @@ export default class RouteLayout extends Component<Props> {
     // order to see items 1 - 4).
     const extraPage = +(routeOffset % PAGE_SIZE > 0)
     const activePage = Math.floor(routeOffset / PAGE_SIZE) + 1 + extraPage
+    const numPages = Math.ceil(numRoutes / PAGE_SIZE)
+
+    const paginationItems = []
+    for (let pageNum = 1; pageNum <= numPages; pageNum++) {
+      paginationItems.push(
+        <Pagination.Item
+          active={pageNum === activePage}
+          key={pageNum}
+          onClick={(e) => this._onPaginationSelect(+e.target.text)}
```

Comment:
```suggestion
          onClick={() => this._onPaginationSelect(pageNum)}
```
I'm not too familiar with what `+e.target.text` does. Maybe this is more readable?

Reply:
`e` is just a general shorthand for the event in the event handler, which in this case is the onClick event that happens when you click on the pagination menu. We need to use that to retrieve the page number that is clicked, otherwise pageNum would pass the current page number. `e.target.text` is just how we can access the text (page number) that was clicked.